contracts/mock/MockUserMintableERC721.sol

```diff
@@ -178,7 +181,10 @@ contract MockUserMintableERC721 is UserMintableERC721 {
       singingRootProof,
```

Comment:
MockUserMintableERC721 is used to test each utility function in isolation in ERC721Document.test.js. I recommend you start with that. To run only the test one want you can use something like ```npx run truffle test test/erc721/ERC721Document.test.js```. Notice `npx` and not `npm' in order run the locally installed truffle and not the global one.